docs/contribution-guide.md

```diff
@@ -26,3 +26,87 @@ Foam is open to contributions of any kind, including but not limited to code, do
 [recipes]: recipes "Recipes"
 [recommended-extensions]: recommended-extensions "Recommended Extensions"
 [//end]: # "Autogenerated link references"
+
+## Contributing to the VS Code Extension
+
+If you're interested in contributing to the VS Code extension (aka `foam-vscode`), this guide will help you get things set up locally.
+
+1. Clone the repo locally:
+
+   `git clone https://github.com/foambubble/foam.git`
+2. Install the necessary dependencies by running this command from the root:
+
+   `yarn install`
+3. This project uses [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/).`foam-vscode` relies on `foam-workspace-manager`. This means we need to compile it before we do any extension development. From the root, run the command:
+
+   `yarn workspace foam-workspace-manager build`
+4. Now we'll use the launch configuration defined at [`.vscode/launch.json`](https://github.com/foambubble/foam/blob/master/.vscode/launch.json) to start a new extension host of VS Code. From the root, or the `foam-vscode` workspace, press f5.
+5. In the new extension host of VS Code that launched, open a Foam workspace (e.g. your personal one, or a test-specific one created from foam-template). This is strictly not necessary, but the extension won't auto-run unless it's in a workspace with a `.vscode/foam`.json file.
+6. Test a command to make sure it's working as expected. Open the Command Palette (Ctrl/Cmd + Shift + P) and select "Foam: Update Markdown Reference List". If you see no errors, it's good to go!
+
+### Tutorial: Adding a New Command
```

Comment:
I wonder if adding new command tutorial needs to be this front and center, since it's not something we expect people to have to do on a regular basis? In the spirit of atomic zettels/bubbles, I'd maybe extract this to its own markdown file and link it from here?

Reply:
Good question! My thinking - try adding a new command as a way to verify that your local development env is set up correctly?

Your call! Would be happy to extract it. Let me know!

Reply:
Actually, why don't I just do that in this PR as you say!